extensions/EXT/EXT_texture_query_lod.txt

```diff
@@ -0,0 +1,186 @@
+Name
+
+    EXT_texture_query_lod
+
+Name Strings
+
+    GL_EXT_texture_query_lod
+
+Contact
+
+    Gert Wollny (gert wollny 'at' collabora.com)
+
+Contributors
+
+    Pat Brown, NVIDIA
+    Greg Roth, NVIDIA
+    Eric Werness, NVIDIA
+
+Notice
+
+    Copyright (c) 2019 Collabora LTD 
+    Copyright (c) 2009-2013 The Khronos Group Inc. Copyright terms at
+        http://www.khronos.org/registry/speccopyright.html
+
+Status
+
+    FIXME
+
+Version
+
+    Last Modified Date:         04/02/2019
+    Revision:                   1
+    Based on ARB_texture_query_lod version 4, modified 2013/10/04.
+
+Number
+
+    OpenGL ES extension #310
+
+Dependencies
+
+    OpenGL ES 3.0 is required.
+
+    OpenGL Shading Language 300 ES is required
+
+    EXT_gpu_shader4 is required.
+
+    EXT_texture_array is required.
+
+    This extension interacts trivially with ARB_texture_cube_map_array
+
+    This extension is written against the OpenGL ES 3.2 specification and
+    version 3.00 ES of the OpenGL Shading Language Specification.
+
+Overview
+
+    This extension adds is based on 
```

Comment:
This looks like an editing error of some sort - maybe the "adds is based on" should be removed and the next line joined up with this one?

Reply:
Fixed